docs/src/main/paradox/routing-dsl/directives/parameter-directives/parameters.md

```diff
@@ -29,47 +25,47 @@ The signature shown is simplified and written in pseudo-syntax, the real signatu
 The parameters directive filters on the existence of several query parameters and extract their values.
 
 Query parameters can be either extracted as a String or can be converted to another type. The parameter name
-can be supplied either as a String or as a Symbol. Parameter extraction can be modified to mark a query parameter
+is supplied as a String. Parameter extraction can be modified to mark a query parameter
 as required, optional, or repeated, or to filter requests where a parameter has a certain value:
 
 `"color"`
 : extract value of parameter "color" as `String`
 
-`"color".?`
-: extract optional value of parameter "color" as `Option[String]`
+`"color".optional`
```

Comment:
Here we should still show the symbolic names as well (if possible with the definition list syntax).

Reply:
It is shown in the text below. Adding it to the same line looked very ugly/